### PR TITLE
add master-events tag

### PR DESCRIPTION
**Pull request by ToasterSticks**
This PR adds the master-events tag

### CODE REVIEW

**Comment by Nekoyre** [comment]
Can you add "master events" as a keyword? Other than that lgtm

File: tags/tags.toml [hunk 1]
```diff
@@ -803,3 +803,18 @@ Version 13 is still in development, the branch moves quickly.
 • [Preview update guide](<https://deploy-preview-680--discordjs-guide.netlify.app/additional-info/changes-in-v13.html>) (use `CTRL` + `F` to search for the old method or property)
 *\\🚧 This is still a draft and may not be up to date at all times*
 """
+
+[master-events]
+keywords = ["master-events", "me", "v13-events", "v13 events"]
```
Comment (thehairy) [inline tags/tags.toml:808]:
```suggestion
keywords = ["master-events", "master events", "me", "v13-events", "v13 events"]
```

**Review by thehairy — CHANGES_REQUESTED** [review]

File: tags/tags.toml [hunk 2]
```diff
@@ -803,3 +803,18 @@ Version 13 is still in development, the branch moves quickly.
...
+On master branch, `message` and `interaction` events have been renamed to `messageCreate` and `interactionCreate` respectively, to bring the library in line with Discord's naming conventions.
+
+Don't worry - the old names still work, but you'll receive a Deprecation Warning until you switch over.
+
+```diff
+- <Client>.on('message', message => { ... });
++ <Client>.on('messageCreate', message => { ... });
+- <Client>.on('interaction', interaction => { ... });
++ <Client>.on('interactionCreate', interaction => { ... });
+```
```
Comment (iCrawl) [inline tags/tags.toml:819]:
```suggestion
The `message` and `interaction` events have been renamed to `messageCreate` and `interactionCreate` respectively, to bring the library in line with Discord's naming conventions.

The old events still work, but you'll receive a deprecation warning until you switch over.

```diff
- client.on('message', message => { ... });
+ client.on('messageCreate', message => { ... });
- client.on('interaction', interaction => { ... });
+ client.on('interactionCreate', interaction => { ... });
```
```

File: tags/tags.toml [hunk 3]
```diff
@@ -803,3 +803,18 @@ Version 13 is still in development, the branch moves quickly.
 • [Preview update guide](<https://deploy-preview-680--discordjs-guide.netlify.app/additional-info/changes-in-v13.html>) (use `CTRL` + `F` to search for the old method or property)
 *\\🚧 This is still a draft and may not be up to date at all times*
 """
+
+[master-events]
+keywords = ["master-events", "master events", "me", "v13-events", "v13 events"]
```
Comment (iCrawl) [inline tags/tags.toml:808]:
```suggestion
keywords = ["master-events", "master events", "v13-events", "v13 events"]
```

Def. remove the `me` one here, too ambigious

File: tags/tags.toml (hunk 2 above)
Comment (anandre) [reply tags/tags.toml:819]: Wouldn't this fall under using the guide interaction to point users to the v13 changes guide?

**Review by SpaceEEC — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Comment by almostSouji** [comment]
i'm not too sure i like the version specific tags, or the <somebranch> keywords in general
i feel like deprecated events and surrounding name and keywords would be much nicer

File: tags/tags.toml [hunk 4]
```diff
@@ -803,3 +803,17 @@ Version 13 is still in development, the branch moves quickly.
 • [Preview update guide](<https://deploy-preview-680--discordjs-guide.netlify.app/additional-info/changes-in-v13.html>) (use `CTRL` + `F` to search for the old method or property)
 *\\🚧 This is still a draft and may not be up to date at all times*
 """
+
+[master-events]
+keywords = ["master-events", "master events", "v13-events", "v13 events"]
```
Comment (thehairy) [inline tags/tags.toml:808]:
```suggestion
keywords = ["event-deprecation", "event deprecation", "messageCreate", "interactionCreate"]
```

**Review by thehairy — CHANGES_REQUESTED** [review]
I agree with souji. Should named version-less.

File: tags/tags.toml [hunk 5]
```diff
@@ -803,3 +803,17 @@ Version 13 is still in development, the branch moves quickly.
 • [Preview update guide](<https://deploy-preview-680--discordjs-guide.netlify.app/additional-info/changes-in-v13.html>) (use `CTRL` + `F` to search for the old method or property)
 *\\🚧 This is still a draft and may not be up to date at all times*
 """
+
+[master-events]
```
Comment (thehairy) [inline tags/tags.toml:807]:
```suggestion
[event-deprecation]
```

File: tags/tags.toml [hunk 6]
```diff
@@ -803,3 +803,17 @@ Version 13 is still in development, the branch moves quickly.
 • [Preview update guide](<https://deploy-preview-680--discordjs-guide.netlify.app/additional-info/changes-in-v13.html>) (use `CTRL` + `F` to search for the old method or property)
 *\\🚧 This is still a draft and may not be up to date at all times*
 """
+
+[event-deprecation]
+keywords = ["event-deprecation", "event deprecation", "master-events", "master events", "v13-events", "v13 events"]
```
Comment (thehairy) [inline tags/tags.toml:808]:
```suggestion
keywords = ["event-deprecation", "event deprecation", "messageCreate", "interactionCreate"]
```

**Review by thehairy — CHANGES_REQUESTED** [review]
Thats what we wanted to get rid of 🧹

**Review by thehairy — APPROVED** [review]

File: tags/tags.toml [hunk 7]
```diff
@@ -803,3 +803,17 @@ Version 13 is still in development, the branch moves quickly.
...
+The `message` and `interaction` events have been renamed to `messageCreate` and `interactionCreate` respectively, to bring the library in line with Discord's naming conventions.
+
+The old events still work, but you'll receive a deprecation warning until you switch over.
+
+```diff
+- client.on('message', message => { ... });
++ client.on('messageCreate', message => { ... });
+- client.on('interaction', interaction => { ... });
++ client.on('interactionCreate', interaction => { ... });
+"""
```
Comment (NotSugden) [inline tags/tags.toml:819]: missing the closing backticks 🤣

**Comment by ToasterSticks** [comment]
Since I butchered this one hard, I re-did it in #12 :mmLol:

**Pull request closed without merging**